src/localization/fi.json

```diff
@@ -5,7 +5,9 @@
     "close": "Sulje",
     "back": "Takaisin",
     "login": "Kirjaudu sisään",
-    "webPage": "Lisätietoja verkkosivuillamme"
+    "webPage": "Lisätietoja verkkosivuillamme",
+    "create": "Create",
```

Comment:
Related to previous comment... If this is create, then the translation is "Luo" if it is reserve then it should be "Varaa"